### PR TITLE
decrement terra delegations in progress by 3 in upgrade

### PR DESCRIPTION
**Pull request by riley-stride**
// Decrement DelegationChangesInProgress on Terra vals by 3
// - Fetches terra host zone
// - Loops validators
// - Decrements each validator's DelegationChangeInProgress by 3

### CODE REVIEW

File: app/upgrades/v18/upgrades.go [hunk 1]
```diff
@@ -67,6 +68,39 @@ func UpdateRedemptionRateBounds(ctx sdk.Context, k stakeibckeeper.Keeper) {
...
+	terra_chainid := "phoenix-1"
+	hostZone, found := sk.GetHostZone(ctx, terra_chainid)
+	if !found {
+		return types.ErrHostZoneNotFound
+	}
+
+	// iterate the validators
+	for _, val := range hostZone.GetValidators() {
+
+		// subtract 3, flooring at 0
```
Comment (riley-stride) [inline app/upgrades/v18/upgrades.go:90]: check me that we should floor at 0

File: app/upgrades/v18/upgrades.go [hunk 2]
```diff
@@ -67,6 +68,39 @@ func UpdateRedemptionRateBounds(ctx sdk.Context, k stakeibckeeper.Keeper) {
...
+// - Fetches terra host zone
+// - Loops validators
+// - Decrements each validator's DelegationChangeInProgress by 3
+func DecrementTerraDelegationChangesInProgress(
+	ctx sdk.Context,
+	sk stakeibckeeper.Keeper,
+) error {
+
+	// grab the terra host zone
+	terra_chainid := "phoenix-1"
```
Comment (riley-stride) [inline app/upgrades/v18/upgrades.go:81]: check me that this should be hardcoded

File: app/upgrades/v18/upgrades_test.go [hunk 3]
```diff
@@ -257,3 +259,47 @@ func (s *UpgradeTestSuite) TestUpdateUnbondingRecords() {
...
+	// Create list of validators
+	validators := []*types.Validator{}
+	for i := 0; i < 5; i++ {
+		address := fmt.Sprintf("val-%d", i)
+		validators = append(validators, &types.Validator{Address: address, DelegationChangesInProgress: int64(i)})
+	}
+
+	// set the host zone
+	hostZone1 := stakeibctypes.HostZone{
+		ChainId:    actualChainId,
```
Comment (sampocs) [inline app/upgrades/v18/upgrades_test.go:276]: after pulling ID to constants, use `v18.TerraChainId`

**Review by sampocs — COMMENTED** [review]
beautiful, just a couple nits!

File: app/upgrades/v18/upgrades.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (riley-stride) [reply app/upgrades/v18/upgrades.go]: done

**Pull request merged**